### DISCUSSION TITLE
Changing material colors

### DISCUSSION THREAD

**Discussion by ndkindt** [Taiizor/ReaLTaiizor#48] [Q&A] [answered]
I'm digging the controls, and some of them are nearly perfect.

One thing that I'm struggling to figure out is how to change accent colors on controls, especially the Material Drawer control.  Right now it's stuck on a hot pink that doesn't match colors anywhere else in my project.  I've played around with the MaterialManager but all I can find are read only properties.  Has anyone had any luck tweaking this?

**Comment by Taiizor** [comment, marked as answer]
Below is an example of how to change the colors.
To put it briefly, you can change the colors using the MaterialManager. However, the colors offered cannot be changed.

https://github.com/Taiizor/ReaLTaiizor/blob/develop/demo/ReaLTaiizor.UI/ReaLTaiizor.UI/Form17.cs

**Reply by ndkindt** [reply]
Thank you!  I don't know how I missed ColorScheme when I was going through my options, but this was exactly what I was looking for.

**Reply by Taiizor** [reply]
nP, i wish you good work 👍🏻

**Reply by Chaugechen** [reply]
I'm sorry! I've been researching for a while, but I'm still unclear about how to change the font color and button color in the demo UI. For example, the most basic button colors: pink and white, and text colors: black and white :(

**Reply by Taiizor** [reply]
You cannot customize individual components' colors because color customizations are associated with the theme. Instead, you can specify text colors using the MaterialSkinManager. Examples can be found at the following links:

https://github.com/Taiizor/ReaLTaiizor/blob/develop/demo/ReaLTaiizor.UI/ReaLTaiizor.UI/Form17.cs#L32
https://github.com/Taiizor/ReaLTaiizor/blob/develop/demo/ReaLTaiizor.UI/ReaLTaiizor.UI/Form17.cs#L33
https://github.com/Taiizor/ReaLTaiizor/blob/develop/demo/ReaLTaiizor.UI/ReaLTaiizor.UI/Form17.cs#L34

**Reply by Chaugechen** [reply]
Çok teşekkür ederim

**Reply by Taiizor** [reply]
Rica ederim (You're welcome)

**Reply by Chaugechen** [reply]
![image](https://github.com/Taiizor/ReaLTaiizor/assets/49526083/24b7baad-1985-42b4-bfbf-437dd30c8b19)
I think I fully understand now; this is a global deployment, and it's not possible to make changes to a single object.

**Reply by Taiizor** [reply]
Yes, but you can also customize the colors yourself.

`new MaterialColorScheme("#00480157", "#370142", "DC2EFF", "00BB5FCF", "#FFFFFF");`
`new MaterialColorScheme(Color.Pink, Color.HotPink, Color.LightPink, Color.DeepPink, Color.DeepPink);`
`new MaterialColorScheme(Color.Orange, Color.DarkOrange, Color.Orchid, Color.OrangeRed, Color.MediumOrchid);`

**Reply by Chaugechen** [reply]
I still can't understand how in the demo, the button can have three eye colors, namely Indigo, Pink, and White.

I can confirm that it's controlled through the properties Primary and Accent, but how is the White color set? It's really amazing.

    materialSkinManager.ColorScheme = new MaterialColorScheme(MaterialPrimary.Indigo500, MaterialPrimary.Indigo700, MaterialPrimary.Indigo100, MaterialAccent.Pink200, MaterialTextShade.LIGHT);

Thank you for being willing to respond to such a basic question ❤️
